tests/unit_node/process_test.ts

```diff
@@ -237,6 +237,7 @@ Deno.test({
 
 Deno.test({
   name: "process.off signal",
+  ignore: Deno.build.os == "windows",
```

Comment:
👍 

LGTM, do we have an issue for process.off not working on Windows?

Reply:
Dug a little deeper on this, turns out it's a `process.off` issue, it's a more general signal sending issue on windows (https://github.com/denoland/deno/issues/22846)